history.rst

```diff
@@ -5,6 +5,35 @@
 OMERO version history
 =====================
 
+
+5.6.14 (November 2024)
+----------------------
+
+This release includes the following upgrade of the OMERO.server Java components:
+
+- omero-gateway-java 5.10.0
+- omero-blitz 5.8.0
+- omero-server 5.7.0
+- omero-renderer 5.6.0
+- omero-romio 5.7.0
+- omero-common 5.7.0
+- omero-model 5.7.0
+
+
+Improvements include:
+
+- Bump slf4j dependencies. See https://github.com/ome/openmicroscopy/pull/6409.
+- Add new lookup tables See https://github.com/ome/openmicroscopy/pull/6398.
+- Ugrade of Bio-Formats to version 8.0.1.
+
+**Note:** The omero-* java components have been built via Java 11 with the compatibility flag on for Java 8.
```

Comment:
`java` -> `Java` 
Alternatively, this note could be moved right under the paragraph that discussed the component upgrades to reduce the repetition.